cardano_node_tests/tests/tests_conway/test_constitution.py

```diff
@@ -555,3 +557,10 @@ def _check_cli_query():
             ledger_3979 = issues.ledger_3979.copy()
             ledger_3979.message = " ;".join(xfail_ledger_3979_msgs)
             ledger_3979.finish_test()
+
+        # Check new constitution proposal in dbsync
+        reqc.db012.start(url=helpers.get_vcs_link())
+        constitution_db = list(dbsync_queries.query_new_constitution(txhash=action_txid))
```

Comment:
This will fail when db-sync is not available. We need to check `configuration.HAS_DBSYNC` if using queries directly in tests.